tools/check-node-fixtures

```diff
@@ -56,6 +56,8 @@ def get_event_checker(
     name = "check_" + event["type"]
     if "op" in event:
         name += "_" + event["op"]
+    if "operation" in event:
+        name += "_" + event["operation"]
```

Comment:
This might become unnecessary when we do the PR to add `op` to the fields where we have `operation`; I think @orientor is working on it?  Might be better to just do that than add this hack.

Reply:
It’s better than adding three things to an exempt list, I think.

Reply:
Seems like @orientor has pushed a PR for this, so I'm pulling this commit.